tests/admin_views/tests.py

```diff
@@ -725,6 +725,11 @@ def test_disallowed_to_field(self):
         response = self.client.get(reverse('admin:admin_views_referencedbyinline_changelist'), {TO_FIELD_VAR: 'name'})
         self.assertEqual(response.status_code, 200)
 
+        # #25622 - Specifying a field of model refered by generic relation shouldn't raise an exception.
```

Comment:
I'd try to be a bit more specific and state the expected behavior like "should raise DisallowedModelAdminToField". Hopefully all of Django doesn't crash. :-)